reproducibility_project/src/engines/mcccs/conv_traj.py

```diff
@@ -0,0 +1,113 @@
+"""Reads the .xyz files for different seeds in NPT MC simulations, and combines and saves them in the hdf5 format."""
```

Comment:
Why is it writing to hdf5 format?  Are the hdf5 files converted to gsd elsewhere?

Reply:
Also tagging @justinGilmer and @jennyfothergill to see what they think about this.

For each system, we have 16 independent seeds. And for each seed I have broken production into 4 parts. So I have a total of 64 trajectories for each system.
If we want to find _g(r)_ for a system, it does not make sense to find _g(r)_ for each replica, instead a Total _g(r)_ of the combined 64 trajectories must be computed. 
This script just reads the traj for each independent seed (and we have 4 prod trajs for each seed) and combines the 64 trajs into one traj for that system which can be used to compute _g(r)_and other properties. 

hdf5 format is optimized for `mdtraj`. In my case one xyz/pdb file takes  around 3 mins to load. We have 64 of them so it takes me 3 hrs to load one complete traj for one system. On the other hand, hdf5 format is very very fast. So I submit a cluster job to load the 64 xyz trajs and save the combined trajectory for each system in the hdf5 format which can be used later to do the _g(r)_ analysis. 

I have not tested how efficient gsd traj is, but I will check that and let you all know.

Reply:
Seems fair, hdf5 is a very open standard. And I am using it when making the job.data stores for the subsampled data.

I am curious to see the comparison between gsd traj too.